src/internal/packager2/inspect.go

```diff
@@ -86,23 +89,25 @@ func handleSBOMOptions(ctx context.Context, pkg v1alpha1.ZarfPackage, opt ZarfIn
 	if err != nil {
 		return err
 	}
-	if opt.SBOMOutputDir != "" {
-		out, err := layout.SBOMs.OutputSBOMFiles(opt.SBOMOutputDir, pkg.Metadata.Name)
+
+	sbomDirPath := opt.SBOMOutputDir
+	if sbomDirPath == "" {
```

Comment:
Why did you choose this path and not to evaluate `opt.SBOMOutputDir` here and then pass through tmpDir in your if below

Reply:
I think this solution remvoes code duplication as we only need a tmpdir when sbom output diir is not set. The previous solution did not require a tmpdir either way which is why i flipped the logic.